onegov/election_day/formats/common.py

```diff
@@ -142,3 +142,44 @@ def load_csv(
         )
 
     return csv, error
+
+
+# Verification utils
+def line_is_relevant(line, number, district=None):
+    if district:
+        return line.sortwahlkreis == district and line.sortgeschaeft == number
+    else:
+        return line.sortgeschaeft == number
+
+
+def validate_integer(line, col, none_be_zero=True):
```

Comment:
I'd say `non_be_zero` should be named `treat_none_as_zero`. What do you think?

Reply:
it's better like this, I also added a default kwarg to return that default and changed the lines where the votes are received for the candidate.